pages/thank-you.js

```diff
@@ -44,6 +44,22 @@ function ThankYou() {
 
   const [cookies, removeCookie] = useCookies(["uuid"]);
 
+  const pullRequests = Object.values(router.query);
+
+  const listItems = pullRequests.map((number) => (
```

Comment:
This is very elegant code, great use of `Array.map` @nathanfletcher 🙌

Reply:
Victor, I will print this out and hang it on my wall. Thank you 🙏🏾.